Lib/uuid.py

```diff
@@ -58,6 +58,7 @@
 _DARWIN  = platform.system() == 'Darwin'
 _LINUX   = platform.system() == 'Linux'
 _WINDOWS = platform.system() == 'Windows'
+_MAC_DELIM =  b':' if not _AIX else b'.'
```

Comment:
Add:

    _MAC_OMITS_LEADING_ZEROES = _AIX

Reply:
This is unnecessary with the current implementation.